functions.php

```diff
@@ -27,5 +28,143 @@
   require_once($filepath);
 }
 
+
+
+
+
+
+
+
+add_action( 'tgmpa_register', 'my_theme_register_required_plugins' );
```

Comment:
Want to throw this over in [`lib/init.php`](https://github.com/SandersForPresident/WordPress/blob/develop/lib/init.php) where the other hooks are?